src/queryResult/queryResultWebViewController.ts

```diff
@@ -100,6 +100,33 @@ export class QueryResultWebviewController extends ReactWebviewViewController<
                 message.selection,
             );
         });
+        this.registerRequestHandler("copySelection", async (message) => {
+            // sendActionEvent();
```

Comment:
Can you add telemetry to all copy handlers? You can follow the `Save As` ones in this same file @cssuh